scripts/claude-context-helper.js

```
@@ -0,0 +1,181 @@
+  run() {
+    const command = process.argv[2];
+    
+    switch (command) {
+      case 'info':
+        this.displayContextInfo();
+        break;
+      
+      case 'title':
+        const title = this.getClaudeTitle();
+        if (title) {
+          console.log(title);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)